contracts/utils/Create2Factory.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+contract Create2Factory {
```

Comment:
Suggest to add some necessary comment, especially, this is deployed in genesis block. Generally, user do not need to deploy such contract.